Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
Pirms commito, atliec atpakaļ uz `https` un noņem to `insecure` no 25. rindiņas, jo manā galā ar to nestrādā. Jāpaskatās, kāpēc tev vajag viņus.